content/docs/deployment/csmoperator/drivers/_index.md

```diff
@@ -1,97 +1,99 @@
+---
+title: "CSI Drivers"
+linkTitle: "CSI Drivers"
+description: Installation of Dell CSI Drivers using Dell CSM Operator
+weight: 1
+---
+
+## Pre-requisites for installation of the CSI Drivers
+
+On Upstream Kubernetes clusters, ensure that to install
+* VolumeSnapshot CRDs - Install v1 VolumeSnapshot CRDs
+* External Volume Snapshot Controller
+
+### Volume Snapshot CRD's
+The Kubernetes Volume Snapshot CRDs can be obtained and installed from the external-snapshotter project on Github. Manifests are available [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v6.2.1/client/config/crd)
+
+### Volume Snapshot Controller
+The CSI external-snapshotter sidecar is split into two controllers:
+- A common snapshot controller
+- A CSI external-snapshotter sidecar
+
+The common snapshot controller must be installed only once in the cluster irrespective of the number of CSI drivers installed in the cluster. On OpenShift clusters, the common snapshot-controller is pre-installed. In the clusters where it is not present, it can be installed using `kubectl` and the manifests are available [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v6.2.1/deploy/kubernetes/snapshot-controller)
+
+*NOTE:*
+- The manifests available on GitHub install the snapshotter image:
+    - [quay.io/k8scsi/csi-snapshotter:v6.2.1](https://quay.io/repository/k8scsi/csi-snapshotter?tag=v6.2.1&tab=tags)
+- The CSI external-snapshotter sidecar is still installed along with the driver and does not involve any extra configuration.
+
+### Installation example
+
+You can install CRDs and the default snapshot controller by running the following commands:
+```bash
+git clone https://github.com/kubernetes-csi/external-snapshotter/
+cd ./external-snapshotter
+git checkout release-<your-version>
+kubectl create -f client/config/crd
+kubectl create -f deploy/kubernetes/snapshot-controller
+```
+*NOTE:*
+- It is recommended to use 6.2.1 version of snapshotter/snapshot-controller.
+
+## Installing CSI Driver via Operator
+
+Refer [PowerScale Driver](../drivers/powerscale) to install the driver via Operator <br>
+Refer [PowerFlex Driver](../drivers/powerflex) to install the driver via Operator <br>
+Refer [PowerStore Driver](../drivers/powerstore) to install the driver via Operator <br>
+
+>**Note**: If you are using an OLM based installation, example manifests are available in `OperatorHub` UI.
+You can edit these manifests and install the driver using the `OperatorHub` UI.
+
+### Verifying the driver installation
+Once the driver `Custom Resource (CR)` is created, you can verify the installation as mentioned below

```

Comment:
Change "as mentioned below" to "as follows"

Reply:
done